server/models/Request.js

```diff
@@ -0,0 +1,35 @@
+const mongoose = require("mongoose");
+
+const requestSchema = new mongoose.Schema({
+    user_id:{
+        type: String,
+        required: true,
+      },
+    sitter_id: {
+        type: String,
+        required: true,
+    },
+    start: {
+        type: Date,
+        required: true
+    },
+    end:{
+        type: Date,
+        required: true
+    },
+    accepted:{
+        type: Boolean,
+        default: False
+    },
+    declined: {
+        type: Boolean,
+        default: False
+    },
```

Comment:
These two fields could definitely do the job but more often than not, the logic for keeping these in a valid state (e.g. both `accepted` and `declined` aren't `true` at the same time) become awkward real quick. So I recommend reducing these to a single field (e.g. `status`) with an [enum validator](https://mongoosejs.com/docs/validation.html#built-in-validators).

Reply:
Thanks, I changed this to have only one field, "status", but considering this is a boolean type I don't think is necessary to include an enum validator. Could you please confirm?

Reply:
Even though you can represent 3 different states with a `Boolean` type (`true/false/null` -> `accepted/declined/pending` respectively), this isn't very intuitive or scalable (e.g. won't be easy to add more `status` values in the future such as `draft` or `completed`). So I recommend to change this field to be a `String` type and use an `enum` validator.

Reply:
make sense